vocabularies/Org.OData.Capabilities.V1.xml

```diff
@@ -692,6 +711,36 @@ supported:
         </Property>
       </ComplexType>
 
+      <Term Name="ReadRestrictions" Type="Capabilities.ReadRestrictionsType" AppliesTo="EntitySet Singleton FunctionImport">

```

Comment:
Please remove ReadRestrictions and ReadRestrictionsType here, it's a duplicate of https://github.com/oasis-tcs/odata-vocabularies/pull/35/files#diff-40dc44aa44d41fa1bf22798bb68972a4R775

Reply:
sorry, my merge from master was bad.